src/install.sh

```diff
@@ -6,10 +6,14 @@ set -o pipefail
 apk update
 
 # install tools
-apk add gnupg python3 py3-pip aws-cli curl
+apk add gnupg aws-cli curl
 
 # install go-cron
-curl -L https://github.com/odise/go-cron/releases/download/v0.0.6/go-cron-linux.gz | zcat > /usr/local/bin/go-cron
+
+curl -L https://github.com/ivoronin/go-cron/releases/download/v0.0.5/go-cron_0.0.5_linux_${TARGETARCH}.tar.gz -O
```

Comment:
This changed repo last commit was 4 years ago 🤔 Maybe we should consider moving to something maintained like https://github.com/go-co-op/gocron.

Reply:
and the one before was from 11 years ago 😂